src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -288,6 +306,14 @@ export async function startTransformByQ() {
         vscode.commands.executeCommand('setContext', 'gumby.isTransformAvailable', true)
         const durationInMs = new Date().getTime() - startTime.getTime()
 
+        // Note: IntelliJ implementation of ResultStatusMessage includes additional metadata such as jobId.
+        telemetry.codeTransform_totalRunTime.emit({
+            codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
+            codeTransformResultStatusMessage: resultStatusMessage.toString(),
+            codeTransformRunTimeLatency: durationInMs,
+            result: MetadataResult.Pass,
```

Comment:
Do we want to set result to `Fail` if an error was thrown? Since `finally` block will always execute

Reply:
sure, will change